.circleci/config.yml

```diff
@@ -209,3 +209,21 @@ workflows:
           filters:
             branches:
               only: main
+      - deploy-graphql-vm:
+          context:
+            - swirrl-ons-datahost-gcloud
+          requires:
+            - deploy-graphql-service
+          env: ons
+          filters:
+            branches:
+              only: ons
+      - deploy-ldapi-vm:
+          context:
+            - swirrl-ons-datahost-gcloud
+          requires:
+            - deploy-ld-openapi-service
+          env: ons
+          filters:
+            branches:
+              only: ons
```

Comment:
There is still a `deploy-ldapi-vm` job configured for the `dev` environment that triggers from the `main` branch.

https://github.com/Swirrl/datahost-prototypes/blob/982468a91d6b0260602001c5a8277dbd94a19a52/.circleci/config.yml#L203

This seems to contradict Ric's request here: https://github.com/Swirrl/datahost-prototypes/issues/254#issuecomment-1735850786

🤔 Or are did you simply choose to do the "future work" now and kill two birds with one stone?

It just looks to me like both branches will currently trigger deploys, when I was only expecting the `ons` branch to trigger deploys for the next few weeks. Perhaps I've misread this, though.

Reply:
Yes, this will configure two environments in the `swirrl-ons-datahost` project. The domain for the current `dev` environment is being changed to `graphql-dev.gss-data.org.uk` and the new `ons` environment is configured with the old `graphql-prototype.gss-data.org.uk`. Commits to the `main` branch deploy to the `dev` environment as they currently do, and commits on the `ons` branch are deployed to the new `ons` environment.

We'll have to re-create the dev environment in a new gcloud project shortly but it was done this way to avoid destroying the existing environment while the new one was being created.